compiler/ballerina-lang/src/main/java/io/ballerina/projects/util/FileUtils.java

```diff
@@ -164,6 +173,45 @@ public static boolean isValidPng(Path filePath) throws IOException {
         return isMatchingImageFormat(filePath, PNG_HEX_HEADER, 8);
     }
 
+    /**
+     * Get last modified timestamp of a ballerina project.
+     *
+     * @param projectRoot project root path
+     * @return last modified time of the ballerina project
+     */
+    public static long lastModifiedTimeOfBalProject(Path projectRoot) {
+        File[] files = projectRoot.toFile().listFiles();
```

Comment:
It's always good to convert it to the absolute path. This might produce a wrong value if the project is executed outside the package directory.